consensus/state_processing/src/per_epoch_processing/altair/inactivity_updates.rs

```diff
@@ -26,7 +26,7 @@ pub fn process_inactivity_updates<T: EthSpec>(
         } else if state.is_in_inactivity_leak(spec) {
             state
                 .get_inactivity_score_mut(index)?
-                .safe_add_assign(INACTIVITY_SCORE_BIAS)?;
+                .safe_add_assign(spec.inactivity_score_bias)?;
```

Comment:
Agree that this should be on the ChainSpec rather than a constant, as per: https://github.com/ethereum/eth2.0-specs/blob/48e1ef193a3c640ddd3df264f25470989908f1c7/configs/minimal_config.yaml